pkgs/development/libraries/crypto++/default.nix

```
@@ -1,49 +1,37 @@
-{ fetchFromGitHub, stdenv }:
+{ stdenv, fetchFromGitHub }:
 
 stdenv.mkDerivation rec {
   pname = "crypto++";
-  majorVersion = "5.6";
-  version = "${majorVersion}.5";
+  version = "8.2.0";
+  majmin = builtins.head ( builtins.match "([[:digit:]]+\.[[:digit:]]+)\.*" version );
+  underscoredVersion = stdenv.lib.strings.replaceStrings ["."] ["_"] version;
 
   src = fetchFromGitHub {
     owner = "weidai11";
     repo = "cryptopp";
-    rev = "CRYPTOPP_5_6_5";
-    sha256 = "1yk7jyf4va9425cg05llskpls2jm7n3jwy2hj5jm74zkr4mwpvl7";
+    rev = "CRYPTOPP_${underscoredVersion}";
+    sha256 = "01zrrzjn14yhkb9fzzl57vmh7ig9a6n6fka45f8za0gf7jpcq3mj";
   };
 
-  patches = stdenv.lib.concatLists [
-    (stdenv.lib.optional (stdenv.hostPlatform.system != "i686-cygwin") ./dll.patch)
-    (stdenv.lib.optional stdenv.hostPlatform.isDarwin ./GNUmakefile-darwin.patch)
-  ];
-
-
-  configurePhase = ''
-      sed -i GNUmakefile \
-        -e 's|-march=native|-fPIC|g' \
-        -e '/^CXXFLAGS =/s|-g ||'
-  '';
-
+  makeFlags = [ "PREFIX=${placeholder "out"}" ];
```

since the patch was removed
```suggestion
  nativeBuildInputs = lib.optional stdenv.isDarwin libtool;
  makeFlags = [ "PREFIX=${placeholder "out"}" ];
```
```
/nix/store/5ys0r8acw7kx5daia5nshycrw0wwnn1g-bash-4.4-p23/bin/bash: libtool: command not found
make: *** [GNUmakefile:1281: libcryptopp.a] Error 127
builder for '/nix/store/yfz1wfwswwi17md6c3ipbzfh1mnb5wmd-crypto++-8.2.0.drv' failed with exit code 2
error: build of '/nix/store/yfz1wfwswwi17md6c3ipbzfh1mnb5wmd-crypto++-8.2.0.drv' failed
```

---

I'm not sure if there is a better way between adding `libtool` to nativeBuildInputs and patching the makefile.
The former is less verbose.